client/my-sites/activity/activity-log-banner/style.scss

```diff
@@ -109,4 +109,39 @@
 
 .activity-log-banner__upgrade {
 	margin-top: 1px;
+
+	.button {
+		white-space: nowrap;
+	}
+}
+
+// Introduction banner
+.activity-log-banner__intro {
+	display: flex;
+	align-items: flex-start;
+	margin-bottom: 44px;
+}
+
+.activity-log-banner__intro-image {
+	max-width: 220px;
+	margin: 8px 16px 0 -8px;
+
+	@include breakpoint( '<960px' ) {
+		max-width: 100px;
+	}
+
+	@include breakpoint( '<660px' ) {
+		max-width: 64px;
+	}
+
+	@include breakpoint( '<480px' ) {
+		display: none;
+	}
+}
+
+.activity-log-banner__intro-description {
+
+	p {
+		font-size: 14px;
```

Comment:
Do you mean between the banners that appear on this page, or elsewhere in Calypso?

Reply:
With banners elsewhere in Calypso — my assumption was that font sizes are "framework" type thing just like colours and should come from variables?
Totally not a biggie to have it like this either though. :-)

Reply:
I don't have a strong preference on this one. While I agree with @simison on this, I agree there can be exceptions - as long as there are conscious decisions behind them. I'm leaving the final call to @keoshi's judgement.

Reply:
Pasting this from a conversation @tyxla and I just had in Slack: my intention was to bump it up a notch on the intro banner since I personally find 13px so small for something that is this important, but I’m not sure of the inconsistencies it might create elsewhere. I still think it's fine to move as is, and adjust later on.

Reply:
Sounds good!